### PR TITLE
feat: CLI table aliases, row-level controllers, and other tweak

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary

- Adds `aliases` flag to CLI docs & examples.
- Adds walkthrough for row-level table ACLs with smart contracts.
- Adds walkthrough for parsing registry events that are emitted when some smart contract constructor/method makes calls to the registry.

### CODE REVIEW

File: docs/smart-contracts/serving-nft-metadata.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ By default, the `tokenURI()` method will take this URI and append the requested
 
 ```tsx
 tokenURI(1) { return _baseURIString + '/' + 1 }
-// https://testnets.tableland.network/query?unwrap=true&extract=true&statement=/1
+// https://testnets.tableland.network/query?unwrap=true&extract=true&statement=1
```
Comment (joewagner) [inline docs/smart-contracts/serving-nft-metadata.md:66]: @dtbuchholz this is obviously correcting a typo, but also noticing that the path is wrong here (maybe that's on purpose?).
should `https://testnets.tableland.network/query` be `https://testnets.tableland.network/api/v1/query`?

Comment (joewagner) [reply docs/smart-contracts/serving-nft-metadata.md:66]: We can address that in a different PR, just figured I'd mention it.

**Review by joewagner — APPROVED** [review]
Tons of great updates here!
🚀

File: docs/smart-contracts/serving-nft-metadata.md (hunk 1 above)
Comment (dtbuchholz) [reply docs/smart-contracts/serving-nft-metadata.md:66]: good catch, updated in my next docs batch!

**Pull request merged**